src/model/Card.ts

```diff
@@ -39,7 +39,7 @@ export class ChainCard {
       card.Health = parseInt(content[cardType].Health);
       card.Attack = parseInt(content[cardType].Attack);
       card.Delay = parseInt(content[cardType].Delay);
-      card.RulesTexts = content[cardType].RulesTexts;
+      card.RulesTexts = [];
```

Comment:
Let's completely remove it from the card type.

Reply:
hmm I think it is being used by the CardCreator to put the rulesText together when you are assembling abilities?